toml-to-ical/src/lib.rs

```diff
@@ -987,7 +987,7 @@ struct RecurrenceRule {
 
 impl fmt::Display for RecurrenceRule {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "RRULE:FREQ={};", self.frequency)?;
+        write!(f, "RRULE:FREQ={}", self.frequency)?;
         if let Some(interval) = &self.interval {
             write!(f, "{interval}")?;
```

Comment:
```suggestion
            write!(f, ";{interval}")?;
```
I think I'd prefer if we put the `;` here rather than in the types.

Reply:
Ohh lol, I've implemented like this but pushed to origin :facepalm:. Fixed now.